hack/preview.sh

```diff
@@ -50,13 +69,11 @@ yq e ".sharedSecret=\"${SHARED_SECRET:-$(openssl rand -hex 20)}\"" $ROOT/compone
     yq e ".serviceProviders[0].type=\"${SPI_TYPE:-GitHub}\"" - | \
     yq e ".serviceProviders[0].clientId=\"${SPI_CLIENT_ID:-app-client-id}\"" - | \
     yq e ".serviceProviders[0].clientSecret=\"${SPI_CLIENT_SECRET:-app-secret}\"" - > $TMP_FILE
+oc create namespace spi-system --dry-run=client -o yaml | oc apply -f -
 oc create -n spi-system secret generic shared-configuration-file --from-file=config.yaml=$TMP_FILE --dry-run=client -o yaml | oc apply -f -
 echo "SPI configured"
 rm $TMP_FILE
 
-# set backend route for quality dashboard for current cluster
-$ROOT/hack/util-set-quality-dashboard-backend-route.sh
-
```

Comment:
why is this not needed anymore?

Reply:
It's disabled in development overview, it requires settings which are not implemented. https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/bootstrap-cluster.sh#L120-L129